applicationinsights.dev.json

```diff
@@ -42,6 +42,17 @@
           }
         ],
         "percentage": 0
+      },
+      {
+        "telemetryType": "exception",
+        "attributes": [
+          {
+            "key": "exception.type",
+            "value": "java.util.concurrent.CompletionException",
+            "matchType": "strict"
+          }
+        ],
+        "percentage": 10
```

Comment:
need to drastically reduce these as these will be logged twice for the majority of cases. 
There's a minor edge case where these wouldn't be logged at all without which is the reason we're not reducing to 0%